lib/chef-dk/policyfile/chef_server_cookbook_source.rb

```diff
@@ -15,28 +15,41 @@
 # limitations under the License.
 #
 
+require 'json'
```

Comment:
Saw it [still being used here](https://github.com/chef/chef-dk/blob/ca00f402aef82b6d67a00962b15e8e6186474179/lib/chef-dk/policyfile/community_cookbook_source.rb#L18) but I have since changed it.